environments.json

```diff
@@ -2978,18 +2978,38 @@
     "family": "V8",
     "short": "CH 110",
     "release": "2023-02-07",
-    "obsolete": false
+    "obsolete": true
   },
   "chrome111": {
-    "full": "Chrome 111 Beta",
+    "full": "Chrome 111",
     "family": "V8",
     "short": "CH 111",
-    "unstable": true
+    "release": "2023-03-01",
+    "obsolete": true
   },
   "chrome112": {
-    "full": "Chrome 112 Canary",
+    "full": "Chrome 112",
     "family": "V8",
     "short": "CH 112",
+    "release": "2023-03-29",
+    "obsolete": true
+  },
+  "chrome113": {
+    "full": "Chrome 113",
+    "family": "V8",
+    "short": "CH 113",
+    "obsolete": false
```

Comment:
I think that it could be good to add a release date.

https://chromestatus.com/roadmap